### PR TITLE
Fix a minor grammar nit, update UI tests

### PR DESCRIPTION
**Pull request by jamesmunns**
Minor fix, but I noticed it while debugging some code

### CODE REVIEW

File: src/test/ui/coherence/coherence-projection-conflict-orphan.re.stderr [hunk 1]
```diff
@@ -3,11 +3,11 @@ error[E0119]: conflicting implementations of trait `Foo<i32>` for type `i32`:
    |
 LL | impl Foo<i32> for i32 { }
    | --------------------- first implementation here
-LL | 
+LL |
```
Comment (jonas-schievink) [inline src/test/ui/coherence/coherence-projection-conflict-orphan.re.stderr]: Just checked locally, this whitespace is significant

**Comment by jamesmunns** [comment]
I think this whitespace is a mistake itself (calling `draw_col_separator()` instead of `draw_col_separator_no_space()` for these lines, but I don't immediately know how to fix this on a wider scale. Restored the whitespace for now.

**Pull request merged**